v-next/hardhat/test/internal/cli/helpers/getGlobalHelpString.ts

```diff
@@ -7,34 +7,33 @@ import { buildGlobalOptionDefinitions } from "@ignored/hardhat-vnext-core";
 import { globalOption, ArgumentType } from "@ignored/hardhat-vnext-core/config";
 import { readClosestPackageJson } from "@ignored/hardhat-vnext-utils/package";
 
+import { BUILTIN_GLOBAL_OPTIONS_DEFINITIONS } from "../../../../src/internal/builtin-global-options.js";
 import { getGlobalHelpString } from "../../../../src/internal/cli/helpers/getGlobalHelpString.js";
 
 describe("getGlobalHelpString", async function () {
   const packageJson = await readClosestPackageJson(import.meta.url);
 
-  describe("when there are no tasks", function () {
+  describe("when there are no tasks or global options", function () {
     it("should return the global help string", async function () {
       const tasks = new Map();
-      const help = await getGlobalHelpString(tasks, new Map());
+      const globalOptionDefinitions = new Map();
+      const help = await getGlobalHelpString(tasks, globalOptionDefinitions);
 
       const expected = `Hardhat version ${packageJson.version}
 
 Usage: hardhat [GLOBAL OPTIONS] <TASK> [SUBTASK] [TASK OPTIONS] [--] [TASK ARGUMENTS]
 
 GLOBAL OPTIONS:
```

Comment:
I believe we should consider this a bug in `getGlobalHelpString`, as it shows the GLOBAL OPTIONS section even when there are no global options. In reality, this will never happen since we always have the built-in global options, so it's probably not a high-priority bug.

Reply:
agreed

Reply:
It's mostly a weird artifact of testing the helper function instead of the higher level end-to-end thing though